.github/workflows/release-cli-sdk.yaml

```diff
@@ -0,0 +1,28 @@
+name: CLI SDK Release
```

Comment:
I don't think there's any "CLI" involved here. This is the TypeScript SDK (communicates with Paladin via HTTP).